### PR TITLE
Add info tooltips to logs that have been sampled by adaptive logs

### PR DESCRIPTION
**Pull request by MasslessParticle**
This PR adds additional context to log lines that have been sampled by Adaptive Logs. This is necessary to help users understand why they may be seeing gaps in their log

### CODE REVIEW

**Review by matyax — COMMENTED** [review]
Just one adjustment to the icon.

**Comment by matyax** [comment]
@MasslessParticle Do you mean for this to show up everywhere? Everywhere = Explore, dashboards, and apps.

**Comment by MasslessParticle** [comment]
@matyax Yeah, it'd be great if this existed everywhere. The hope is to give as specific of feedback as possible when things are getting dropped

File: public/app/features/logs/components/getLogRowStyles.ts [hunk 1]
```diff
@@ -120,6 +120,11 @@ export const getLogRowStyles = memoizeOne((theme: GrafanaTheme2) => {
     logIconError: css`
       color: ${theme.colors.warning.main};
```
Comment (matyax) [inline public/app/features/logs/components/getLogRowStyles.ts:121]: Would you mind doing the same change (position relative, top: -2px, etc) to the `logIconError` please? Thanks!

**Review by matyax — APPROVED** [review]
LGTM!

**Comment by MasslessParticle** [comment]
Added the styles to the warning

**Review by matyax — APPROVED** [review]

**Pull request merged**